packages/clay-nav/src/Nav.tsx

```diff
@@ -11,32 +11,60 @@ import {NavLink} from './NavLink';
 
 interface IProps extends React.HTMLAttributes<HTMLUListElement> {
 	/**
-	 * Flag to indicate if `nav-nested` class should be applied.
+	 * Flag to indicate if `nav-pills` class should be applied. Adds background colors and border styles to navigation items.
+	 */
+	navPills?: boolean;
```

Comment:
I think you might've missed these props

Reply:
Oh, you gave a 👍 to that [here](https://github.com/liferay/clay/pull/3207#pullrequestreview-409504327) so I thought those were okay, sure, I'll remove them.